unity-environment/Assets/ML-Agents/Examples/PushBlock/Scripts/GoalDetect.cs

```diff
@@ -0,0 +1,25 @@
+﻿//Detect when the orange block has touched the goal. 
+//Detect when the orange block has touched an obstacle. 
+//Put this script onto the orange block. There's nothing you need to set in the editor.
+//Make sure the goal is tagged with "goal" in the editor.
+
+using UnityEngine;
+
+public class GoalDetect : MonoBehaviour {
+	[HideInInspector]
```

Comment:
save tabs as spaces to fix indentation problems (in several files)

Reply:
Reformatted the push block files w/ spaces.